docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: '3'
```

Comment:
Perhaps we should have a `docker` directory, with `docker/docker-compose-kafka.yaml` inside, so that we can split out other cases if needed. (And document how to specify it with `docker-compose up`) Any thoughts?

Reply:
For now, I'd keep it simple, a single `docker-compose.yaml` at the root of the project.
This way you don't have to specify the file, and you can still lunch a single "service" by doing:
`docker-compose up redis` or  `docker-compose kafka`.

Reply:
I've added a `start` script, which is used like:
```
./scripts/start  # starts everything
./scripts/start kafka # only kafka
./scripts/start postgres # only postgres
./scripts/start redis  # only redis
```

Reply:
Oh ace, that sounds good.